packages/actionmenu/src/react/item.tsx

```diff
@@ -1,21 +1,25 @@
-import { compose, css } from 'glamor'
-import PropTypes from 'prop-types'
+import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
+import { StyleAttribute, compose, css } from 'glamor'
 import React, {
+  HTMLAttributes,
+  KeyboardEventHandler,
+  MouseEventHandler,
   useState,
   useRef,
   forwardRef,
   useImperativeHandle,
-  useContext,
-  HTMLAttributes
+  useContext
 } from 'react'
 
-import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
-import stylesheet from '../css/index'
-import { tagName } from '../vars/index'
+import stylesheet from '../css'
+import { tagName } from '../vars'
+
 import { ActionMenuContext } from './context'
 import { Arrow } from './arrow'
 
-const styles = {
+type StyleFn = (obj?: unknown) => StyleAttribute
```

Comment:
```suggestion
type StyleFn = (props?: ItemProps) => StyleAttribute
```

Would this not allow you to avoid retyping all the objects (eg, line 23)?